extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -76,21 +76,13 @@ private function getInsightModule( $type, $config ) {
 		 * communitypage-uncategorizedpages-title'
 		 * communitypage-wantedpages-title'
 		 * communitypage-deadendpages-title'
-		 * communitypage-popularpages-description'
-		 * communitypage-uncategorizedpages-description'
 		 * communitypage-wantedpages-description'
-		 * communitypage-deadendpages-description'
 		 */
 		$insightPages['type'] = $type;
 		$insightPages['title'] = wfMessage( 'communitypage-' . $type . '-title' )->text();
-		$insightPages['description'] =  wfMessage( 'communitypage-' . $type . '-description' )->text();
-
-		// TODO: to be removed in the scope of: https://wikia-inc.atlassian.net/browse/WW-83
-		$insightPages['mediumtitle'] = wfMessage( 'communitypage-' . $type . '-mediumtitle' )->text();
-		$insightPages['hightitle'] = wfMessage( 'communitypage-' . $type . '-hightitle' )->text();
-		$insightPages['mediumdescription'] =  wfMessage( 'communitypage-' . $type . '-mediumdescription' )->text();
-		$insightPages['highdescription'] =  wfMessage( 'communitypage-' . $type . '-highdescription' )->text();
-
+		$insightPages['description'] =  $type === 'wantedpages' ?
+			wfMessage( 'communitypage-wantedpages-description' )->text() :
+			null;
 
 		if ( $insightPages['count'] > self::INSIGHTS_MODULE_ITEMS ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)